src/core/lv2/Lv2Manager.cpp

```diff
@@ -90,53 +82,38 @@ Lv2Manager::Lv2Manager() :
 	supportOpt(Lv2UridCache::Id::bufsz_sequenceSize);
 }
 
-
-
-
 Lv2Manager::~Lv2Manager()
```

Comment:
Space holder is now 1 lines instead of 4. Hope this is OK...

Reply:
I'm okay with 1 line.

Reply:
I would prefer 2 lines to have a bit higher visual separation, but I agree that 1 line is better than 4.

Reply:
No further complaints, resolving this.